sky/cli.py

```diff
@@ -378,7 +377,7 @@ def _install_shell_completion(ctx: click.Context, param: click.Parameter,
         cmd = '_SKY_COMPLETE=fish_source sky > \
                 ~/.config/fish/completions/sky.fish'
 
-        reload_cmd = _RELOAD_FISH_CMD
+        reload_cmd = None
```

Comment:
Would be good to add a comment about the behavior of fish as described in the PR description. : )